### PR TITLE
feat: Adds "AFTER" constraint at createColumnConstraint

### PR DESCRIPTION
**Pull request by FabioBachi**
Adds the possibility to use the AFTER constraint in the addColumn methods for MySQL.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ var MysqlDriver = Base.extend({
     );
     return {
       foreignKey: constraint.foreignKey,
-      constraints: [escapedName, t, len, constraint.constraints].join(' ')
+      constraints: [escapedName, t, len, constraint.constraints].join(' '),
```
Comment (wzrdtales) [inline index.js:96]: please comfort your auto fixer with db-migrates internal rules:

https://github.com/db-migrate/mysql/blob/master/.prettierrc
https://github.com/db-migrate/mysql/blob/master/.eslintrc

trailing commas are not part of this for example.

**Review by wzrdtales — CHANGES_REQUESTED** [review]
in general ok just my comments and the missing requirement of a test for this new functionality

**Comment by wzrdtales** [comment]
there are now proper githooks in place, please sync with the master branch and install the dependencies

**Comment by FabioBachi** [comment]
> in general ok just my comments and the missing requirement of a test for this new functionality

Hi! I have just mande the changes and updated a test to validate this new feature.

**Review by wzrdtales — APPROVED** [review]

**Comment by wzrdtales** [comment]
thank you for your contribution :tada: !

**Comment by mjfold** [comment]
Hey @wzrdtales when can we expect this to get an npm release?

**Comment by wirdehall** [comment]
> Hey @wzrdtales when can we expect this to get an npm release?

I would also like to know this. Can we get any news about when the next release is planed? :)

**Comment by wzrdtales** [comment]
sorry this got under the radar again version comes today

**Pull request merged**